types/three/src/animation/tracks/NumberKeyframeTrack.d.ts

```diff
@@ -2,7 +2,7 @@ import { KeyframeTrack } from './../KeyframeTrack';
 import { InterpolationModes } from '../../constants';
 
 export class NumberKeyframeTrack extends KeyframeTrack {
-    constructor(name: string, times: any[], values: any[], interpolation?: InterpolationModes);
+    constructor(name: string, times: number[], values: any[], interpolation?: InterpolationModes);
```

Comment:
oo, `ArrayLike<number>` and `number[]` are different since `number[]` does not accept `Float32Array`.

This is the definition of `ArrayLike<T>`:
https://github.com/microsoft/TypeScript/blob/acf854b636e0b8e5a12c3f9951d4edfa0fa73bcd/lib/lib.es5.d.ts#L1554-L1557

I believe `values` is also `ArrayLike<number>`.
`BooleanKeyframeTrack` and `StringKeyframeTrack` are exceptions, but otherwise values can be `Float32Array` too.

(and I don't know why it used to be `any[]` which does not accept `Float32Array` ...)

Reply:
Okay than,
All `times` params have to be `ArrayLike<number>` and, `values` have to be `ArrayLike<number>` except `BooleanKeyframeTrack` and `StringKeyframeTrack` right?

(Any way can I ask what is oo means?)

Reply:
yeah, I think that's right.

> (Any way can I ask what is oo means?)

Just an interjection 😄

Reply:
I revised it! 

I hope you've been a great holiday and happy new year :)